requirements.txt

```diff
@@ -2,19 +2,18 @@ aiohttp==3.9.1
 aiohttp-retry==2.8.3
 aiosignal==1.3.1
 alembic==1.12.0
-alembic-utils==0.8.1
+alembic-utils==0.8.2
```

Comment:
this looks like it should also be update in the `requirements-app` file. You might want to double check if there are others.

Reply:
Good catch, Evan.